frontend/src/components/Dropdown.js

```diff
@@ -10,47 +10,45 @@ import './Dropdown.css';
 
 const styles = {
   root: {
-  	paddingLeft: 8,
+    paddingLeft: 8,
   },
 };
 
 class Dropdown extends Component {
-	handleChange = event => {
-		this.props.changeValue(event.target.value);
-	}
-
-	renderMenuItems() {
-		let retval = [];
-
-		this.props.dropdownVals.map( val => {
-			retval.push(
-				<MenuItem value={val}>{val}</MenuItem>
-			);
-		});
-
-		return retval
-	}
-
-	render() {
-		const { classes } = this.props;
-
-		return (
-			<div className="dropdown">
-				<p class="title">{this.props.title}</p>
-				<Select 
-					className="select"
-					value={ this.props.selectValue }
-          onChange={ this.handleChange }
-          classes={{ root: classes.root }}
-				>
-					<MenuItem value="">
-            <em>None</em>
+  handleChange = event => {
+    this.props.changeValue(event.target.value);
+  }
+
+  renderMenuItems() {
+    const sortedNames = Object.keys(this.props.dropdownVals)
```

Comment:
`this.props.dropdownVals` changed from type array to type map so that we can define a value and a name for the value. IE census division id and census division name.